### ISSUE TITLE
self-update fails with release/bundled build

### ISSUE THREAD

**Issue by brandonbloom** [deref/exo#56]
![image](https://user-images.githubusercontent.com/119164/127402495-06ddfa62-2b59-48e9-8676-9d46fda2e905.png)

from exod.log:

```
2021/07/28 14:55:37 error processing request: running update script: fork/exec /var/folders/cn/q2gg_ww95f5249g5xtfm4jp40000gn/T/example827031344: no such file or directory
2021/07/28 14:56:38 error processing request: running update script: fork/exec /var/folders/cn/q2gg_ww95f5249g5xtfm4jp40000gn/T/example214949991: no such file or directory
```

via @siggisim

**Comment by kendru** [comment]
This should be fixed by https://github.com/deref/exo-install/commit/7b4304685290959c73e799b9808877320fc3d3b1 and https://github.com/deref/exo/commit/6e90f0ebce0e243f78672887d5d0c3f795a080bb. Will test on macos and close the issue if it works.

**Comment by kendru** [comment]
Fixed in `2021.07.29`
